### PR TITLE
Cleaned logging for compliance checker

### PR DESCRIPTION
**Pull request by blaumeiser-at-bosch**
#### Description
* Prettified output
* Removed redundant output, especially redundant exception stack traces
* Removed tests for removed functionality
* Moved compliance tooling log levels from info to debug to remove clutter
* Improved error message for license qualified rule

#### Request Reviewer
@neubs-bsi @bs-ondem 

#### Type of Change
- [ ] Bug fix
- [ ] New feature
- [x] Improvements
- [ ] Documentation update
- [ ] Other

#### How Has This Been Tested?
- [ ] Specific Unit Test
- [x] Executing the Example Projects
- [x] `mvn test` of whole project
- [ ] `mvn test` of submodule
- [ ] `gradle test` (for the gradle plugin)

#### Checklist
- [x] My code follows the style and guidelines of this project
- [x] I have self-reviewed my code 
- [ ] I have provided tests that prove my change is working 
- [x] Existing tests still pass with my changes 
- [ ] I have updated the documentation accordingly to my changes

### CODE REVIEW

File: core/runtime/src/main/java/org/eclipse/sw360/antenna/workflow/AntennaWorkflow.java [hunk 1]
```diff
@@ -66,60 +65,43 @@ public AntennaWorkflow(AntennaWorkflowConfiguration antennaWFConfig) {
...
-                .map(c -> "\n        - " + c)
-                .collect(Collectors.joining());
+    private void createResultLines(Collection<String> resultLines, String workflowItemName, Set<IEvaluationResult> evaluationResults) {
+        evaluationResults.stream()
+                .forEach(evalRes ->
+                        resultLines.add(
+                                String.format("%s: %s for artifacts %s%n          %s",
+                                        workflowItemName,
+                                        evalRes.getId(),
+                                        evalRes.getFailedArtifacts().stream().map(Artifact::artifactAsCoordinate).collect(Collectors.joining(", ")),
```
Comment (neubs-bsi) [inline core/runtime/src/main/java/org/eclipse/sw360/antenna/workflow/AntennaWorkflow.java]: After `stream()´ we normally do a new line per following stream function

File: core/runtime/src/main/java/org/eclipse/sw360/antenna/workflow/stubs/AbstractComplianceChecker.java [hunk 2]
```diff
@@ -148,18 +100,18 @@ private void reportResults(IProcessingReporter reporter, Set<IEvaluationResult>
...
-    }
-
-    private String getCoordinates(Artifact a) {
-        Optional<Coordinate> coordinate = a.getCoordinateForType(Coordinate.Types.MAVEN);
-        if (! coordinate.isPresent()) {
-            coordinate = a.getCoordinates().stream().findFirst();
+        String resultString = String.format("Policy Violation: %s (%s): %s%n          %s",
+                result.getId(),
+                result.getSeverity(),
+                result.getFailedArtifacts().stream().map(Artifact::artifactAsCoordinate).collect(Collectors.joining(", ")),
```
Comment (neubs-bsi) [inline core/runtime/src/main/java/org/eclipse/sw360/antenna/workflow/stubs/AbstractComplianceChecker.java]: After `stream()´ we normally do a new line per following stream function

File: core/runtime/src/main/java/org/eclipse/sw360/antenna/workflow/stubs/AbstractComplianceChecker.java [hunk 3]
```diff
@@ -148,18 +100,18 @@ private void reportResults(IProcessingReporter reporter, Set<IEvaluationResult>
...
+                result.getDescription());
```
Comment (neubs-bsi) [inline core/runtime/src/main/java/org/eclipse/sw360/antenna/workflow/stubs/AbstractComplianceChecker.java]: pretty similar to the string-ification in `AntennaWorkflow`. would it be worth it to make a toString() function in EvaluationResult?

Comment (blaumeiser-at-bosch) [reply core/runtime/src/main/java/org/eclipse/sw360/antenna/workflow/stubs/AbstractComplianceChecker.java]: I have made a new proposal, which uses a interface default message, but perhaps this is also not the best solution. It enters output related stuff to a pure data interface 🤔

File: core/model/src/main/java/org/eclipse/sw360/antenna/api/IEvaluationResult.java [hunk 4]
```diff
@@ -49,4 +50,15 @@ public static Severity fromValue(String v) {
      * @return All artifacts that have broken this rule.
      */
     Set<Artifact> getFailedArtifacts();
+
+    default String resultAsMessage() {
+        return String.format("%s (%s): %s%n          %s",
+                getId(),
+                getSeverity(),
+                getFailedArtifacts()
```
Comment (neubs-bsi) [inline core/model/src/main/java/org/eclipse/sw360/antenna/api/IEvaluationResult.java]: Sorry to be picky, but I checked the code and we normally do it like this:
```java
variable.potentialfunction().stream()
    .map(...)
    .collect(...)
```
Though I just checked the code and saw that there is a big inconsistency in formatting... We should maybe set a standard now and then adhere to it.

**Pull request merged**